ios/PurchasesHybridCommon/PurchasesHybridCommon/RCCommonFunctionality.h

```diff
@@ -55,6 +57,8 @@ typedef void (^RCHybridResponseBlock)(NSDictionary * _Nullable, RCErrorContainer
 
 + (void)paymentDiscountForProductIdentifier:(NSString *)productIdentifier discount:(nullable NSString *)discountIdentifier completionBlock:(RCHybridResponseBlock)completion;
 
++ (void)presentCodeRedemptionSheet API_AVAILABLE(ios(14.0)) API_UNAVAILABLE(tvos, macos, watchos);
```

Comment:
this should compile correctly against xcode 11 since we don't call the paymentQueues methods directly from here, we're just calling our own methods